ChangeLog

```diff
@@ -1,3 +1,24 @@
+22.3.1
+ - Identify Huawei Cloud as OpenStack (#1689) [huang xinjie]
```

Comment:
We don't want to include any commits beyond `Fix v2 interface matching when no MAC (LP: #1986551)` as adding content like Identify Huawei cloud is functionally different than what we released to Ubuntu for 22.3.1